cmd/svcat/binding/unbind_cmd.go

```diff
@@ -83,6 +105,43 @@ func (c *unbindCmd) Run() error {
 	var bindings []types.NamespacedName
 	var err error
 
+	if c.abandon {
+		fmt.Fprintln(c.Output, "This action is not reversible and may cause you to be charged for the broker resources that are abandoned.")
+		if !c.skipPrompt {
+			fmt.Fprintln(c.Output, "Are you sure? [y|n]: ")
+			s := bufio.NewScanner(os.Stdin)
+			s.Scan()
+
+			err = s.Err()
+			fmt.Fprintln(c.Output, err)
+
+			if strings.ToLower(s.Text()) != "y" {
+				err = fmt.Errorf("aborted abandon operation")
+				return err
+			}
+		}
+
+		var removedBindings []types.NamespacedName
+		if c.instanceName != "" {
+			si := &v1beta1.ServiceInstance{ObjectMeta: metav1.ObjectMeta{Name: c.instanceName, Namespace: c.Namespace}}
+			removedBindings, err = c.App.RemoveFinalizersByInstance(si)
+		} else {
+			retrievedBindings := c.getBindingsToDelete()
+			removedBindings, err = c.App.RemoveFinalizersForBindings(retrievedBindings)
+		}
+
+		if err != nil {
+			return err
+		}
+		for _, binding := range removedBindings {
+			fmt.Fprintln(c.Output, fmt.Sprintf("deleted %s", binding.Name))
```

Comment:
its possible for both these to print, but the removing finalizers is an if/else, so shouldn't this be as well?

Reply:
actually, looking further, neither of these print statements is needed, we say "deleted foobar down below"

Reply:
@jberkhahn I have a return at the end of the if block `if c.abandon {`, so with the current code it won't actually go down to this and later print the `delete foobar`:
```
	if c.instanceName != "" {
		bindings, err = c.App.Unbind(c.Namespace, c.instanceName)
	} else {
		bindings, err = c.App.DeleteBindings(c.getBindingsToDelete())
	}
```
Should I not return at the end of the block `if c.abandon {`? Is `Unbind` or `DeleteBindings` needed when all finalizers are removed?